src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -368,7 +372,10 @@ private void setup(OptionSet options, Properties properties) {
 		if ( properties != null ) {
 			for (Enumeration<Object> e = properties.keys(); e.hasMoreElements(); ) {
 				String k = (String) e.nextElement();
-				if (k.startsWith("kafka.")) {
+				if (k.startsWith("custom.")) {
+				    this.customProperties.setProperty(k.replace("custom.", ""), properties.getProperty(k));
```

Comment:
maybe `custom_producer.` here?  I dunno.  Something a bit more clear.

Reply:
I was wondering about that, actually. How about `custom_producer_factory` just to be clear as to the expected implementation?

Reply:
I just saw this and I'll make this tweak.